.github/workflows/asset-transparency.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
     steps:
     - name: Gather URLs from GitHub release and publish
       id: asset-transparency
-      uses: transparencylog/github-releases-asset-transparency-verify-action@v10
+      uses: transparencylog/github-releases-asset-transparency-verify-action@c77874b4514ae4003994ece9582675195fe012e2 # v11
```

Comment:
Looks like you have a commit here you should move out of your PR
<img width="757" alt="image" src="https://github.com/helm/helm/assets/4860914/caa7c6d1-12df-4d7c-b15e-d74415b1aaca">

Reply:
I can tell this is going to take me longer to do than the original PR as I haven't done this in a while and things have changed on Github.

This was of course added on the original PR by dependabot when I created the PR - I can't remove this from my upstream remote (helm/helm branch of the PR) with a force push of course. Waiting to see if the @ command triggeres a recreate off my branch but I think the only option I'm sure of is to close the PR and reopen with a new branch name at this point.

I'll need to come back to this when I have time to dig around it more.

Reply:
OK, confusion overcome, I didn't see the commits in my local branch until I rebased again. Now cleaned up.